src/objects/zcl_abapgit_object_iwsg.clas.abap

```diff
@@ -98,4 +102,48 @@ CLASS zcl_abapgit_object_iwsg IMPLEMENTATION.
     get_generic( )->serialize( io_xml ).
 
   ENDMETHOD.
+
+  METHOD get_rule.
+    DATA ls_record TYPE /iwfnd/i_med_srh.
+    DATA ls_item TYPE zif_abapgit_rule=>ty_item.
+    DATA lt_item TYPE zif_abapgit_rule=>ty_items.
+
+    SELECT SINGLE *
```

Comment:
My intention of selecting the DB is to retain the creation date/user if the object is modified multiple times. If the select is not used, it will always use current user/date for the creation date which might be an issue. What do you think?

Reply:
Renamed the class/interface. Also, removed the DB select to keep the logic simple for this exception. However, please check my above comment and let me know your thoughts.

Reply:
We do not retain "create by/at" of the source system anywhere else. Typically, deserialize will set those to the current user. In some cases, deserialize will preserve this info when updating the object but in general that's not implemented.